client/Components/Map.js

```diff
@@ -10,17 +10,19 @@ const Wrapper = styled.div`
 
 `
 
-const Map = () => {
+const Map = ({locationData}) => {
+
+  const coordinates = [locationData.coordinates[1], locationData.coordinates[0]]
```

Comment:
Assuming this is latitude/longitude, it might be worth creating an issue with Emmanuel to use standard order.

Reply:
good call